src/SymbolServer.jl

```diff
@@ -28,6 +28,7 @@ function getstore(ssi::SymbolServerInstance, environment_path::AbstractString, p
     server_script = joinpath(@__DIR__, "server.jl")
 
     env_to_use = copy(ENV)
+    ENV["JULIA_REVISE"] = "manual" # Try to make sure Revise isn't enabled.
```

Comment:
```suggestion
    env_to_use["JULIA_REVISE"] = "manual" # Try to make sure Revise isn't enabled.
```
We want this set for the symbol server process, right? Then it needs to go into `env_to_use`.